packages/rollup/src/rollup.config.js

```diff
@@ -0,0 +1,8 @@
+// Note: we cannot require() any plugins here because the user may not have any installed
+
+module.exports = {
+  output: {
+    name: 'bundle',
```

Comment:
so by default if you don't specify a custom config you get an output dir named after the rule and a `bundle.js` in there?

Reply:
I'll add a test for default config, it's missing